autodoc/data/admin-api.lua

```diff
@@ -60,6 +60,28 @@ return {
         care should be taken when setting up Kong environments to avoid undue public
         exposure of this API. See [this document][secure-admin-api] for a discussion
         of methods to secure the Admin API.
+
+        The most of the Kong admin API is auto-generated based on entity schemas,
```

Comment:
This description is great, but I think it should be in the plugin developer guide, not the Admin API reference (for a Kong end-user who never customizes Kong, this is an internal implementation detail; for a Kong plugin author who creates entities, this is super relevant — this also makes me think that we should at one point auto-generate Admin API references for the custom plugin entities in the documentation pages for plugins (heads up @Kong/team-docs!)). I suggest we open a PR in docs.konghq.com with this section added to the Plugin development guide.

Reply:
The idea was that when user sees this at the beginning, he can in most cases just skip the rest of the admin api docs.

Reply:
@hishamhm, do you think we should still remove it? It is one commit to be removed, so not a big deal.

Reply:
I have a backup of that. And will send a PR later to docs.